xbps-src

```diff
@@ -237,18 +237,18 @@ check_reqhost_utils() {
 
     [ "$IN_CHROOT" ] && return 0
 
-    for f in ${REQHOST_UTILS}; do
-        if ! command -v ${f} &>/dev/null; then
-            echo "${f} is missing in your system, can't continue!" 1>&2
+    for f in "$REQHOST_UTILS"; do
```

Comment:
this, for example, will run the loop only once with the entire value of REQHOST_UTILS, not once per whitespace-separated item in the value